provision/core/composer/provision.sh

```diff
@@ -44,17 +42,19 @@ function composer_setup() {
   # Update both Composer and any global packages. Updates to Composer are direct from
   # the master branch on its GitHub repository.
   vvv_info " * Checking for composer updates"
-  if [[ -n "$(noroot composer --version --no-ansi | grep 'Composer version')" ]]; then
+  if ! noroot composer --version --no-ansi | grep 'Composer version'; then
     vvv_info " * Updating Composer..."
     COMPOSER_HOME=/usr/local/src/composer noroot composer --no-ansi global config bin-dir /usr/local/bin
     COMPOSER_HOME=/usr/local/src/composer noroot composer --no-ansi self-update --2 --stable --no-progress --no-interaction
     vvv_info " * Making sure the PHPUnit 7.5 package is available..."
     COMPOSER_HOME=/usr/local/src/composer noroot composer --no-ansi global require --prefer-dist --no-update --no-progress --no-interaction phpunit/phpunit:^7.5
   fi
 
-  vvv_info " * Updating global composer packages..."
-  COMPOSER_HOME=/usr/local/src/composer noroot composer --no-ansi global update --no-progress --no-interaction
-  vvv_success " * Global composer package update completed"
+  if [ -f /usr/local/src/composer/composer.json ]; then
```

Comment:
On MacOS this folder doesn't exist, it's stored elsewhere. We need to figure out how to do this using composer commands

Reply:
I believe `composer config -g` can be used

Reply:
perhaps this:

```sh
~/dev/vvv ᚴ:develop 
❯ composer config -g data-dir
/Users/tarendai/.composer
```

or the `home` config value

Reply:
there's also `composer global show`

Reply:
I can confirm that on debian:
```$ composer config -g data-dir
/home/mte90/.composer
```

Reply:
This is a non-issue. Even on Mac, the VVV environment itself is Linux, and that path is valid.